src/coreclr/jit/ssabuilder.cpp

```diff
@@ -892,21 +891,17 @@ void SsaBuilder::RenameDef(GenTree* defNode, BasicBlock* block)
             // If necessary, add "lclNum/ssaNum" to the arg list of a phi def in any
             // handlers for try blocks that "block" is within.  (But only do this for "real" definitions,
             // not phi definitions.)
-<<<<<<< HEAD
             GenTree* asgPhiOp;
 #if defined(TARGET_WASM)
             if (block->IsLIR())
             {
-                asgPhiOp = asgNode->gtGetOp1();
+                asgPhiOp = defNode->gtGetOp1();
             }
             else
 #endif
-                asgPhiOp = asgNode->gtGetOp2();
+                asgPhiOp = defNode;
 
-            if (!asgPhiOp->OperIs(GT_PHI))
-=======
-            if (!defNode->IsPhiDefn())
->>>>>>> 1a31bf638c6c220d20ef65f43a07a9ac562d92d9
+            if (!asgPhiOp->IsPhiDefn())
```

Comment:
This is not quite right for `#ifdef TARGET_WASM`, since `asgPhiOp` will be `Data()`, i. e. not the actual definition node.

I think we can simplify the whole thing down to just `!defNode->IsPhiDefn()`:
```diff
-            GenTree* asgPhiOp;
-#if defined(TARGET_WASM)
-            if (block->IsLIR())
-            {
-                asgPhiOp = defNode->gtGetOp1();
-            }
-            else
-#endif
-                asgPhiOp = defNode;
-
-            if (!asgPhiOp->IsPhiDefn())
+            // NativeAOT-LLVM: revert the deletion of STORE_LCL_VAR handling in "IsPhiDefn" when merging https://github.com/dotnet/runtime/pull/76139 (Oct 20)
+            if (!defNode->IsPhiDefn())
             {
                 AddDefToHandlerPhis(block, lclNum, ssaNum);
             }
```